politeiawww/api/v1/v1.go

```diff
@@ -63,6 +63,10 @@ const (
 	RouteUnauthenticatedWebSocket = "/ws"
```

Comment:
Actually I think we need to separate cms and www apis out. This is a bit of a pain in the ass but it makes sense to have them in different files and doco.